packages/colorpickers/src/types/index.ts

```diff
@@ -39,6 +39,13 @@ export interface IRGBColor {
   alpha?: number;
 }
 
+export interface IRGBColorProps {
+  $red: number;
+  $green: number;
+  $blue: number;
+  $alpha?: number;
+}
```

Comment:
Can this move to a `types.ts` file within the `styled` directory? We just wouldn't want to mistake the fact that this interface should never be publicly exported.